File: Heat/ViewModels/Invoices/DeleteInvoiceViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Invoices
    Public Class DeleteInvoiceViewModel
        <Key> _
        Property ID As Integer

        <Display(name:="Numero")> _
        Property InvoiceNumber As String

        <Display(name:="Cliente")> _
        Property CustomerName As String

        <Display(name:="Data documento")> _
        <DisplayFormat(DataFormatString:="{0:d}")> _
        Property InvoiceDate As DateTime

        Property Rows As List(Of PresentationInvoiceRowViewModel)
        ' Property Rows As List(Of InvoiceRowViewModel)

    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/DeleteInvoiceViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Invoices
{
    public class DeleteInvoiceViewModel
	{
		[Key()]
		public int ID { get; set; }

		[Display(Name = "Numero")]
		public string InvoiceNumber { get; set; }

		[Display(Name = "Cliente")]
		public string CustomerName { get; set; }

		[Display(Name = "Data documento")]
		[DisplayFormat(DataFormatString = "{0:d}")]
		public DateTime InvoiceDate { get; set; }

		public List<PresentationInvoiceRowViewModel> Rows { get; set; }
		// Property Rows As List(Of InvoiceRowViewModel)

	}
}

